irohad/simulator/impl/simulator.cpp

```diff
@@ -70,6 +70,7 @@ namespace iroha {
         log_->warn("Could not fetch last block: " + e->error);
         return;
       }
+      block_query_opt = boost::none;
```

Comment:
Use braces instead (aka `{` & `}`) for dealing with lifetimes, it's easier to understand stuff like that